GPU/Common/SplineCommon.cpp

```diff
@@ -361,16 +383,45 @@ static void _SplinePatchFullQuality(u8 *&dest, u16 *indices, int &count, const S
 	}
 }
 
+template <bool origNrm, bool origCol>
+static inline void SplinePatchFullQualityDispatch3(u8 *&dest, u16 *indices, int &count, const SplinePatchLocal &spatch, u32 origVertType, int quality, int maxVertices) {
+	bool origTc = (origVertType & GE_VTYPE_TC_MASK) != 0;
```

Comment:
I'm considering about this code what does it mean.
Why does place this template functions here?
This seems really strange. I think we can write more simply.

@hrydgard Do you know what this mean?